CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.10.2dev
 
+* [Fix] Changed variable name in `./src/sql/plot.py` file (#759)
```

Comment:
the changelog notes are usually for end-users, since this is an internal change, we don't create a changelog entry. I'll add the no-changelog label to this PR so you can delete this and the CI will still pass